source/how-govuk-sign-in-works.html.md.erb

```diff
@@ -1,68 +1,46 @@
-1. GOV.UK Sign In records the user’s choice to accept or reject.
+GOV.UK Sign In uses 2 different environments:
 
-You can read more about [choosing which user attributes your service can request](/integrate-with-integration-environment/choose-which-user-attributes-your-service-can-request/).
+* an integration environment, which contains sample users and data for you to test your service’s integration with GOV.UK Sign In
+* a production environment, which is the live environment for real users to access and use your service’s integration with GOV.UK Sign In
 
-## Generating tokens to get information about your users
+## Understand the flow GOV.UK Sign In uses
 
-GOV.UK Sign In will generate:
+<%= image_tag "top-level-technical-diagram.svg", { :alt => '' } %>
 
-* an ID token, which contains the unique subject identifier (UUID) for your user
-* an access token, which your service can use to access additional attributes (if the user has given permission)
+1. Your service asks the user to sign in or create an account.
+1. If your service needs confidence your user is who they say they are, GOV.UK Sign In will request proof of identity.
+1. GOV.UK Sign In collects evidence of the user’s identity.
+1. GOV.UK Sign In provides information about your user.
 
-<%= image_tag "generating-tokens.svg", { :alt => '' } %>
+You can read [guidance about cookies on GOV.UK](https://www.gov.uk/help/cookie-details) if you want to learn more about cookies for GOV.UK Accounts.
 
+To understand the technical flow, for example the endpoints, requests and tokens, there's a more detailed technical diagram you can use.
 
-1. After getting your user’s consent, GOV.UK Sign In redirects your user back to your service, along with an authorisation code tagged onto the end of the URL.
-1. Your service calls the `/token` endpoint with the authorisation code to get an ID token and an access token.
-1. GOV.UK Sign In returns an ID token and an access token to your service.
-1. Optionally, your service can use the access token to make a call to the `/userinfo` endpoint to access additional attributes if the user has agreed to share these.
 
-## Understand the environments GOV.UK Sign In uses
+## Understand the technical flow GOV.UK Sign In uses
 
-GOV.UK Sign In uses 2 different environments:
+<%= image_tag "technical-flow-diagram.svg", { :alt => '' } %>
 
-* an integration environment, which contains sample users and data for you to test your service’s integration with GOV.UK Sign In
-* a production environment, which is the live environment for real users to access and use your service’s integration with GOV.UK Sign In
+1. To make an authorisation request, your service sends the user to the `/authorize` endpoint.
+1. The user logs in (or creates an account if they do not have one) and proves their identity if your service needs them to. GOV.UK Sign In lets your user know how their data will be shared with your service.
+1. GOV.UK Sign In returns an authorisation code.
```

Comment:
Who receives the authorisation code? 

I initially read this as Sign In returning a code to the user, like a 2FA situation. I'm guessing that it's actually the service receiving the code, though.

Reply:
Perfect!